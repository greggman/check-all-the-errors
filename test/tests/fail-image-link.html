<img src="resources/not-exists.jpg" />
