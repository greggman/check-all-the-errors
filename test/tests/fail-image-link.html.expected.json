{
  "numErrors": 1,
  "pages": {
    "http://localhost:8090/fail-image-link.html": {
      "status": 200,
      "errors": [
        {
          "type": "msg",
          "location": {
            "url": "http://localhost:8090/resources/not-exists.jpg"
          },
          "text": "Failed to load resource: the server responded with a status of 404 (Not Found)"
        }
      ]
    }
  },
  "responses": [
    {
      "href": "http://localhost:8090/fail-image-link.html",
      "status": 200
    },
    {
      "href": "http://localhost:8090/resources/not-exists.jpg",
      "status": 404
    }
  ]
}